cooperator_website/migrations/12.0.3.0.0/post-migration.py

```diff
@@ -0,0 +1,11 @@
+from openupgradelib.openupgrade import migrate
+
+
+@migrate()
+def migrate(env, version):
```

Comment:
You're overriding the definition of `migrate` here. This is discouraged.

the same is also true in `pre-migration.py`.

```suggestion
from openupgradelib import openupgrade


@openupgrade.migrate()
def migrate(env, version):
```

Reply:
Is it different to what was done [here](https://github.com/coopiteasy/vertical-cooperative/blob/12.0-komunigi/cooperator/migrations/12.0.5.0.0/pre-migration.py) for instance ? If yes, could you explain the difference. If no, do you have an example of "good practice migration" ?

Reply:
The difference is this:

```python
from foo import bar

# here, bar = foo.bar

def bar():
    # here, bar = __name__.bar
    return 0

# also here, bar = __name__.bar
```

vs

```python
import foo

# here, bar is undefined, and foo.bar = foo.bar

def bar():
    # here, bar = __name__.bar
    return 0

# also here, bar = __name__.bar
```

The second example is better because:

- It uses very clear namespacing.
- `bar` always has one definition.

You want to avoid overriding variable names as much as possible. For a clearer example, imagine this:

```python
float = 1.5
int = 1

def rounddown(number):
    # ERROR: int is not a function!
    return int(number)

assert rounddown(float) == int
```

Reply:
Great, Thank you for the nice explanation :smile:

Reply:
fixed !